uber/forms/attendee.py

```diff
@@ -28,6 +29,10 @@ def valid_cellphone(form, field):
                               'include a country code (e.g. +44) for international numbers.')
 
 
+class LotteryApplication(MagForm):
```

Comment:
Let's move this into its own forms/hotel.py file, to more closely reflect where it is in models/. I'm sure we'll have plenty of lottery-related forms as we progress so it won't be a file with just one class in it.

Also, just wondering, why is this a StringField and not a SelectMultipleField or similar? Is it because the choices have to be ordered?